src/styles/components/reviews-wrapper.scss

```diff
@@ -0,0 +1,73 @@
+.reviews {
+  display: flex;
+  align-items: center;
+  margin-left: 144px;
+  &-wrapper {
+    margin: 0 324px 212px 96px;
+    &__title {
+      font-family: $fontPlayfair;
+      font-size: $fontSizeXL;
+      mix-blend-mode: normal;
+      font-weight: 700;
+      color: $black;
+      margin: 0 0 0 144px;
+    }
+  }
+  &-about {
+    display: flex;
+    flex-flow: column;
+    margin-right: 170px;
+    &__stars {
+      width: 190px;
+    }
+    &__comment {
+      font-family: $fontMulish;
+      mix-blend-mode: normal;
+      font-size: $fontSizeL;
+      margin-top: 30px;
+      font-weight: 400;
+      line-height: 1.5;
+      color: $black;
+    }
+    &__name {
+      font-family: $fontInter;
+      mix-blend-mode: normal;
+      font-size: $fontSizeM;
+      margin-top: 40px;
+      font-weight: 700;
+      color: $black;
+    }
+    &__text {
+      font-family: $fontInter;
+      mix-blend-mode: normal;
```

Comment:
Group selectors by comma with this property

Reply:
Fixed